### ISSUE TITLE
Stop sending a notification each time an admin edits a dispute form

### ISSUE THREAD

**Issue by marcondag** [debtcollective/parent#198]
Broken out from https://github.com/debtcollective/parent/issues/194

According to Dawn, most admin edits to the dispute forms are to correct typos. The member doesn't need to be notified of every change the admin makes. To remedy this, we'll just remove the code that makes the call to create the discourse message.

## Tasks

- [ ] Remove code that sends notifications after admins make changes to a Dispute

## Acceptance criteria

- When an admin edits a dispute, the user shouldn't get a notification

**Comment by orlando** [comment]
@debtcollective/engineers this was confirmed by @dawn3296, we do need to remove all the notifications sent to the user when an admin edits the dispute form

**Comment by shoshber** [comment]
My understanding based on our call just now: 

1. There is a checkbox allowing the admin to send a notification upon edit, if they choose to. We will not remove this.
2. There are some other user notifications which are entirely automatic (for example, the notification that the user has completed the dispute (not the one with the download link)). We will remove these notifications.

Is that correct?

**Comment by marcondag** [comment]
I am confused by this because we definitely do not send a discourse message when an admin updates the form data, but I've heard Orlando and Dawn both say this?

**Comment by dawn3296** [comment]
Hello..
Regarding removing alerts.
For emails to user and admin when the dispute tool form is edited. Should
be removed.

For example,
I change the $$$ of debt on form and save.
No alert.
If i fix a typo in collector address on form.
No alert.


If i change status....
Yes alert...
.....how every that works with discourse.

**Comment by marcondag** [comment]
I'm taking a look at this and am still confused. Could another developer confirm this for me (@orlando @shoshber). The admin dispute form edit endpoint that gets hit routes to `Admin/DisputesController#updateDisputeData` (`/views/includes/disputes/information-form.pug:L5`). Then that controller method calls the `Dispute.prototype.setForm` (`/models/Dispute.js:L306`) function which does not send any messages. That controller method then saves the dispute and redirects the request to the `admin/disputes/show` view.

**Comment by orlando** [comment]
I'll check the code tomorrow morning, but if we are not sending anything, we can close this ticket

**Comment by marcondag** [comment]
@orlando Did you get a chance to verify this?

**Comment by orlando** [comment]
@marcondag yes and you are right, there's no code for sending a message. I'll close this as invalid.

I didn't see a call to regenerate the zip files after we modify the Dispute either, is this happening somewhere else or we are not doing that when the admin modifies a dispute?

**Comment by shoshber** [comment]
@dawn3296 You said:

> For emails to user and admin when the dispute tool form is edited. Should
> be removed.
> 
> For example,
> I change the $$$ of debt on form and save.
> No alert.
> If i fix a typo in collector address on form.
> No alert.

We believe that alerts were not being sent in this scenario. Is that not correct?
